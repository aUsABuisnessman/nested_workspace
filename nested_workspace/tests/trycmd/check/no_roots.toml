args = ["check"]

[bin]
name = "cargo"

[fs]
cwd = "../../../../fixtures/no_roots"
