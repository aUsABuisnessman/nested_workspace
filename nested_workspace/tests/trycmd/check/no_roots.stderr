...
   Compiling no_roots v0.1.0 ([CWD])
    Finished `dev` profile [unoptimized + debuginfo] target(s) in [..]s
